data/hoverboard.config.json

```diff
@@ -55,5 +55,33 @@
     "lg": {
       "min": "1024px"
     }
-  }
+  },
+  "socialNetwork": [
+    {
+      "name": "facebook",
+      "url": "https://www.facebook.com/devfest.ukraine/"
+    },
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/devfest_ukraine/"
+    },
+    {
+      "name": "instagram",
+      "url": "https://www.instagram.com/devfestukraine"
+    },
+    {
+      "name": "youtube",
+      "url": "https://www.youtube.com/playlist?list=PLt8lEzcLNl31AouwDkbLbLARlQFwNS2en"
+    }
+  ],
+  "gplusShareLink": "https://plus.google.com/share?url=https://devfest.gdg.org.ua/",
+  "blogLink": "https://medium.com/gdg-lviv",
+  "mailto": "devfest@gdg.org.ua",
+  "mailchimp": {
+    "url": "https://gdg.us11.list-manage.com/subscribe/post?u=b7e853a79164ddfdbda3ed77b&amp;id=7993e39fbe",
+    "name": "b_b7e853a79164ddfdbda3ed77b_7993e39fbe"
+  },
+  "gdgLvivUrl": "http://lviv.gdg.org.ua/",
```

Comment:
It should not be hardcoded to GDG Lviv. Try something like `gdgChapterUrl` or extend `organizer` object:
```
"organizer": {
    "name": "GDG Lviv",
    "twitter": "gdglviv",
    "url": "..."
  }
```